app/src/main/java/com/cornellappdev/android/eatery/ui/components/general/EateryCard.kt

```diff
@@ -374,54 +452,117 @@ fun EateryCardPrimaryHeader(eatery: Eatery, isCompact: Boolean) {
                 color = GrayFive,
                 style = EateryBlueTypography.subtitle2
             )
-            EateryMenuSummary(eatery = eatery)
+//            EateryMenuSummary(eatery = eatery)
         }
     }
 }
 
 @Composable
-fun EateryCardSecondaryHeader(eatery: Eatery, isCompact: Boolean) {
-    if (!isCompact && eatery.getWalkTimes() != null) {
-        val walkText =
-            "${if (eatery.getWalkTimes()!! > 0) eatery.getWalkTimes() else "< 1"} min walk"
+fun EateryCardSecondaryHeader(eatery: Eatery, isCompact: Boolean, isGridView: Boolean) {
+    if (!isCompact) {
         Row(
             modifier = Modifier.padding(top = 2.dp),
             verticalAlignment = Alignment.CenterVertically
         ) {
-            Icon(
-                painter = painterResource(id = R.drawable.ic_walk_small),
-                contentDescription = null,
-                tint = GrayFive,
-                modifier = Modifier.padding(end = 4.dp, top = 1.dp)
-            )
+            val openUntil = eatery.getOpenUntil()
             Text(
-                text = walkText,
-                fontSize = 14.sp,
-                fontWeight = FontWeight(500),
-                color = GrayFive,
-                style = EateryBlueTypography.subtitle2
+                modifier = Modifier.padding(top = 2.dp),
+                text =
+                if (openUntil == null) "Closed"
+                else if (eatery.isClosingSoon()) "Closing at $openUntil"
+                else ("Open until $openUntil"),
+                style = EateryBlueTypography.subtitle2,
+                color = if (openUntil == null) Red
+                else if (eatery.isClosingSoon()) Yellow
+                else Green
             )
 
-            val waitTimes = eatery.getWaitTimes()
-            if (!waitTimes.isNullOrEmpty()) {
-                DotSeparator()
-                Icon(
-                    Icons.Default.Schedule,
-                    contentDescription = null,
-                    tint = GrayFive,
-                    modifier = Modifier.padding(end = 4.dp, top = 1.dp)
-                )
-                Text(
-                    text = waitTimes,
-                    color = GrayFive,
-                    style = EateryBlueTypography.subtitle2
-                )
-            }
-
         }
     }
 }
 
+//TODO, integrate backend for the entree recommendation once backend is done for that
+@Composable
+fun EateryCardTertiaryHeader(eatery: Eatery, isCompact: Boolean, isGridView: Boolean) {
```

Comment:
Same thing with this Composable, let's comment out its usages instead of making it empty.